src/txmempool.h

```diff
@@ -672,10 +672,10 @@ class CTxMemPool
 class CCoinsViewMemPool : public CCoinsViewBacked
 {
 protected:
-    CTxMemPool &mempool;
+    const CTxMemPool &mempool;
 
 public:
-    CCoinsViewMemPool(CCoinsView *baseIn, CTxMemPool &mempoolIn);
+    CCoinsViewMemPool(CCoinsView *baseIn, const CTxMemPool &mempoolIn);
```

Comment:
If you don't want to do that manually, you can just do `$ git diff HEAD~ -U0|cfd`. Where `cfd` is an alias for  `./contrib/devtools/clang-format-diff.py -p1 -i -v`.